content/content-tool/tool/src/webapp/vm/content/sakai_resources_list.vm

```diff
@@ -37,7 +37,7 @@
             #else
 			<li class="firstToolBarItem"><span class="current">$clang.getString("list.site")</span></li>
             #end
-			#if($canDeleteResource)
+			#if($canDeleteResource && $showTrash)

```

Comment:
I think "site.soft.deletion" is not related with resources trash, why are you changing the condition to be able to see trash in resources tool?